### ISSUE TITLE
WireShutdown::seasonErrorMessage() should be opt-in

### ISSUE THREAD

**Issue by the-ge** [processwire/processwire-issues#1698]
### Short description of the issue

WireShutdown::seasonErrorMessage() should be opt-in instead of hardcoded, at least for the echoed error messages.

### Expected behavior

I wish my error messages would not contain any extras.

### Actual behavior

My error messages are trying to be empathic ¯\_(ツ)_/¯. I have ChatGPT for that :D

### Optional: Screenshots/Links that demonstrate the issue

![screenshot_2023-03-02-133722](https://user-images.githubusercontent.com/6468893/222419093-24a18aef-7fcf-47fa-b7a0-efdb834bf46c.png)

### Optional: Suggestion for a possible fix

Maybe an additional `$config` property and have WireShutdown::sendErrorMessage check for it, like
```PHP
279	protected function sendErrorMessage($message, $why, $useHTML) {
280		
281		$hadOutput = $this->sendExistingOutput();
282		if($hadOutput) echo "\n\n";
283
284		if($this->config && $this->config->debug && $this->config->seasonDebug) { // $this->config->seasonDebug added here
285			$message = $this->seasonErrorMessage($message);
286		}
287		
288		// return text-only error
289		if(!$useHTML) {

[...]
```
There is only one other call to WireShutdown::seasonErrorMessage(), from WireShutdown::sendFatalEmail(), but seasoning seems to make more sense there.

### Steps to reproduce the issue

1. Insert an error in code.
2. Enjoy an empathic error mesage.

### Setup/Environment

* ProcessWire version: 3.0.213
* (Optional) PHP version: 8.2
* (Optional) MySQL version: MariaDB 10.6
* (Optional) Not important.

**Comment by teppokoivula** [comment]
I do enjoy empathic error messages, so had to downvote this one ;)

Seriously though, if this is purely a matter of preference I'd suggest closing the issue and opening a feature request instead. This is not an obvious bug, malfunction, usability/accessibility issue, etc. and thus does not seem like a very good fit for the issues repository.

I do get that error messages with extra flair are not everyone's cup of tea, but many of us find them either positive, or at the very least completely harmless. In case they do cause actual harm, please do clarify what that would be — it is entirely possible that I'm missing something here!

**Comment by the-ge** [comment]
Well, it's certainly a rather tiny inconvenience. I was on the fence on the issue versus feature request dilemma and there were two reasons for deciding it was an issue:
1. The extra strings make the error message shift all the time. Not an issue unless you reload the page repeatedly. I know, I should get better at debugging, but I only have a hammer, so all my problems are nails until I'll get more tools.
2. The principle thing: it's about removing an extra that was added for a, dare I say, quite subjective reason. Not about adding something.

I'll admit I was curious about how developers think about this kind of issues, although, in the hindsight, would've made more sense to post in the forums.

Meanwhile, I found that I can extend the `WireShutdown` class, override the `sendErrorMessage()` method (same code actually, just commented the three lines calling `seasonErrorMessage()`), and replace the original in /site/boot.php with:
```php
wire()->shutdown = new \TheGe\ProcessWire\WireShutdownOverride($config);
```
and, woohoo, the empathy is stopped!

I'll close the issue and search the forums for the café :grin:
